charts/internal/calico/templates/node/daemonset-calico-node.yaml

```diff
@@ -24,6 +24,7 @@ spec:
   template:
     metadata:
       labels:
+        node.gardener.cloud/critical-component: "true"
```

Comment:
The label needs to be added to the `DaemonSet` metadata as well now, see https://github.com/gardener/gardener/issues/7117#issuecomment-1425346730.